app/lib/finder_api.rb

```diff
@@ -156,20 +127,27 @@ def query_builder_class
     SearchQueryBuilder
   end
 
-  def is_all_content?
-    base_path == "/all-content"
-  end
+  FINDERS_IN_DEVELOPMENT = {
+    "/news-and-communications" => "news_and_communications",
+    "/research-and-statistics" => "statistics",
```

Comment:
Was is a conscious decision to drop `research` from the content item file name? By doing so I wonder if we're risk loosing important context?

Reply:
So I've looked at this some more and can see that in the content item we're not including the document types for the research supertype.

https://github.com/alphagov/govuk_document_types/blob/2645f543bea870b9469a70d6c93fde4db6e493dd/data/supertypes.yml#L468-L472

With that being the case we can't really include `research` in the URL. Also how does this effect what displays on the organisation pages?

Reply:
I updated this and the content item so that this is explicitly a stats finder. There's another card to iterate on the behaviour (https://trello.com/c/40IA4pPe/241-design-research-statistics-finder) before we ship this.